README.md

```diff
@@ -1 +1,38 @@
-# GitHub-Flow
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> In this project, you will set up a "Hello world" repository. No complex coding is required for this exercise. Your goal here is to master all of the tools and best practices you learned about in previous steps. You will be using them in all Microverse projects and most likely in your future job as well, so it is important to know them!
+
+
+## Built With
+
+- HTML, CSS
+- VS Code
+
+
+
+## Author(s)
+
+👤 **Wahaj**
+
+- GitHub: [@Wahaj-Ali](https://github.com/Wahaj-Ali)
+- Twitter: [@Ali96Wahaj](https://twitter.com/Ali96Wahaj)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/wahaj-ali-82b9b1164)
+
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
```

Comment:
- [ OPTIONAL]As required by the previous review your `README` file should be descriptive . AKA: Microverse [README](https://github.com/microverseinc/readme-template/blob/master/README.md) template should be used :wink: